### DISCUSSION TITLE
[Beta Projects] Visibility on heirarchy of issues in table and board view

### DISCUSSION THREAD

**Discussion by jess-williamson-treespleasegames** [community/community#6485] [Projects and Issues]
As a user I want to be able to see which issues are hierarchically connected in the beta project.
This would be very useful in the backlog project view, tasks that exist in a tasklist (subtask) of another issue are indented and grouped beneath their parent issue. 
If this also worked on the board view that would be very helpful for managing our iterations.

**Comment by feerrenrut** [comment]
This seems to be described on the public roadmap, but without a specific timeline (only 'future'): https://github.com/github/roadmap/issues/284

See also (currently the most upvoted discussion I could find): https://github.com/github-community/community/discussions/5714
